frontend/src/metabase/status/components/StatusListing/StatusListing.tsx

```diff
@@ -1,22 +1,17 @@
 import React from "react";
-import Database from "metabase-lib/metadata/Database";
 import DatabaseStatus from "../../containers/DatabaseStatus";
 import FileUploadStatus from "../FileUploadStatus";
 import { StatusListingRoot } from "./StatusListing.styled";
 
 export interface StatusListingProps {
   isAdmin: boolean;
-  database?: Database;
 }
 
-const StatusListing = ({
-  isAdmin,
-  database: uploadDatabase,
-}: StatusListingProps): JSX.Element => {
+const StatusListing = ({ isAdmin }: StatusListingProps): JSX.Element => {
   return (
     <StatusListingRoot>
       {isAdmin && <DatabaseStatus />}
-      {uploadDatabase?.canWrite() && <FileUploadStatus />}
```

Comment:
@npfitz I don't see any reason to check authentication here at all. I think the right place to check permissions is whether the user can upload at all. if that is properly handled, they'll never see this status indicator at all, right?

In any event `.canWrite()` I think means "has sql query permissions", which doesn't quite align with "can upload CSVs" right now.